### DISCUSSION TITLE
Gradients

### DISCUSSION THREAD

**Discussion by elpuas** [system-ui/theme-ui#1011] [General] [answered]
How to use gradients on `export default theme`

**Comment by atanasster** [comment]
@elpuas - not sure what you mean. 
There is a `sx` prop example using `@theme-ui/color` here: https://theme-ui.com/packages/color/#advanced-usage

If you want to have it in the theme by default for a component of yours, you can use the same code in the theme object as well.

**Comment by lachlanjc** [comment, marked as answer]
If you're looking to use a gradient on a component, you can use a functional value inside `sx`, like this:

```js
sx={{
  backgroundImage: theme => `linear-gradient(to bottom, ${theme.colors.red}, ${theme.colors.orange})`
}}
```

Does that answer your question?

**Comment by elpuas** [comment]
Yeah, you're both right, on the example, there was a missing ',' so I was confused, but it is all cool now great product.
